xblock/core.py

```diff
@@ -105,6 +105,20 @@ def open_local_resource(cls, uri):
 
         return pkg_resources.resource_stream(cls.__module__, os.path.join(cls.resources_dir, uri))
 
+    @property
+    def supports_save(self):
```

Comment:
Would it be enough to just add a no-op `save()` call here and drop `supports_save()`?